src/assets/SvgIcons/LocationMarker.tsx

```
@@ -0,0 +1,34 @@
+import SvgIcon from "src/components/shared/SvgIcon";
+
+const LocationMarker = (props: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)